### ISSUE TITLE
the new .pb isn't work well

### ISSUE THREAD

**Issue by yihengCoding** [amir-abdi/keras_to_tensorflow#51]
I do follow the way that Readme showed, but my new .pb is not work as same as .h5,I do not know the reason and also do not know how to fix it...It seems like no matter what image is delievered into the .pb，there is only one same output.

**Comment by Jiaqian-Xu** [comment]
have you solved the problem? I also confused about this issue@yihengCoding

**Comment by 98-Jane** [comment]
have you solved the problem? I also confused about this issue@yihengCoding

**Comment by yihengCoding** [comment]
Yes, I made it. The reason is versons of python, keras, etc. are incompatible. Maybe u can check it for sure. Good luck!

**Comment by yihengCoding** [comment]
@98-Jane

**Comment by 98-Jane** [comment]
A lot of thanks! I will check it~@yihengCoding
